### PR TITLE
Redesign the Region tooltip

### PR DESCRIPTION
**Pull request by Longwelwind**
Some screenshots:

image

image

image

image

### CODE REVIEW

File: agot-bg-game-server/src/client/MapComponent.tsx [hunk 1]
```diff
@@ -111,48 +113,22 @@ export default class MapComponent extends Component<MapComponentProps> {
...
+            {(region.supplyIcons > 0 || region.crownIcons) > 0 && (
+                <>
+                    <br />{region.supplyIcons > 0 && <><b>{region.supplyIcons}</b> barrels</>}
+                    {(region.supplyIcons > 0 && region.crownIcons > 0) && " - "}
+                    {region.crownIcons > 0 && <><b>{region.crownIcons}</b> crowns</>}
+                </>
+            )}
+            {region.units.size > 0 && (
+                <><br/>{joinReactNodes(region.units.values.map(u => <b key={u.id}>{u.type.name}</b>), ", ")}</>
+            )}
```
Comment (norielbit) [inline agot-bg-game-server/src/client/MapComponent.tsx:131]: I feel like some representation for routed units could be nice, such as strikethrough and/or not making them bold. Doesn't solve Issue #488 but will alleviate some urgency off it.

File: agot-bg-game-server/src/client/MapComponent.tsx [hunk 2]
```diff
@@ -111,48 +113,22 @@ export default class MapComponent extends Component<MapComponentProps> {
...
-            </table>
+            <b>{region.name}</b> {controller && (<small>of <b>{controller.name}</b></small>)} {region.castleLevel > 0 && (<small> ({region.castleLevel == 1 ? "Castle" : "Stronghold"})</small>)}
+            {region.superControlPowerToken ? (
+                <small><br/>Capital of {region.superControlPowerToken.name} {region.garrison > 0 && <>(Garrison of <b>{region.garrison}</b>)</>}</small>
+            ) : (
+                region.garrison > 0 && (<small><br />Neutral force of <b>{region.garrison}</b></small>)
+            )}
+            {(region.supplyIcons > 0 || region.crownIcons) > 0 && (
+                <>
+                    <br />{region.supplyIcons > 0 && <><b>{region.supplyIcons}</b> barrels</>}
```
Comment (norielbit) [inline agot-bg-game-server/src/client/MapComponent.tsx:124]: I feel like 'barrels' and 'crowns' should be capitalized, but better verify it with someone who has better English than me.

File: agot-bg-game-server/src/client/MapComponent.tsx (hunk 1 above)
Comment (Longwelwind) [reply agot-bg-game-server/src/client/MapComponent.tsx:131]: Good idea! I'll add that.

File: agot-bg-game-server/src/client/MapComponent.tsx (hunk 2 above)
Comment (Longwelwind) [reply agot-bg-game-server/src/client/MapComponent.tsx:124]: It also feels weird, but I don't know. I'll capitalize them, we'll see how it works out.

**Pull request merged**